### ISSUE TITLE
Move Meeting time

### ISSUE THREAD

**Issue by mhdawson** [nodejs/diagnostics#184]
If possible can we move the meeting to be 1 hour earlier?

#nodejs/diagnostics

**Comment by joyeecheung** [comment]
I am OK with that. Although when we have a TSC meeting stretching longer than 1 hour there would be conflicts.

The at-mention didn't work so @nodejs/diagnostics

**Comment by mike-kaufman** [comment]
Yes, let's go ahead and move.  The 11 AM PDT time is consistent with the discussion we had on this just prior to the DST adjustment.  There have not been any objections when discussed for last week's meeting.  @mhdawson  - can you update the recurring meeting on the [nodejs calendar](https://calendar.google.com/calendar/embed?src=nodejs.org_nr77ama8p7d7f9ajrpnu506c98%40group.calendar.google.com)?  I don't  have perms.  

If anyone objects to this time move, please speak up & we'll try to accommodate.

**Comment by mhdawson** [comment]
Meeting moved, thanks.
